src/layouts/layout.module.scss

```
@@ -9,12 +9,16 @@ body {
   background-image: url("../../static/images/banner-blog/banner-desktop.png");
   background-repeat: no-repeat;
   background-size: contain;
+  width: 100%;
+  height: 63.6rem;
 }
 
 .homeBanner {
   background-image: url("../../static/images/home-banner/home-banner-desktop.png");
   background-repeat: no-repeat;
   background-size: contain;
+  width: 100%;
+  height: 63.6rem;
```

homeBanner and blogBanner have some styles in common, maybe you can define these styles at the same time:
 ```suggestion
 .homeBanner,
 .blogBanner {
  width: 100%;
  height: 63.6rem;
  background-repeat: no-repeat;
  background-size: contain;
```